src/openrct2/interface/Viewport.cpp

```diff
@@ -1665,25 +1665,40 @@ void get_map_coordinates_from_pos(
 void get_map_coordinates_from_pos_window(
     rct_window* window, int32_t screenX, int32_t screenY, int32_t flags, int16_t* x, int16_t* y, int32_t* interactionType,
     TileElement** tileElement, rct_viewport** viewport)
+{
+  ScreenCoordsXY screenCoords(screenX, screenY);
+  CoordsXY mapCoords();
```

Comment:
`{}` or just don't put anything. At present it is trying to interpret this as a function definition.